packages/react-vapor/src/components/verticalLine/styles/VerticalLine.scss

```diff
@@ -0,0 +1,9 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
+@import '~coveo-styleguide/scss/variables.scss';
+
+.separator {
```

Comment:
Why is it called sepator here but the component is called `VerticalLine` ? I would tend to rename one or the other so that both the css class and the react component have the same name.